modules/atomic-widgets/module.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Modules\AtomicWidgets\Widgets\Atomic_Heading;
+use Elementor\Plugin;
+use Elementor\Widgets_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+	const EXPERIMENT_NAME = 'atomic_widgets';
+
+	const PACKAGES = [
+		'editor-documents', // TODO: NEED to be removed once the editor will not be dependent on the documents package.
+		'editor-panels',
+	];
```

Comment:
Where is the new package? `editor-editing-panel`?

Reply:
Done.